### DISCUSSION TITLE
Resource for curated repository of CSL styles

### DISCUSSION THREAD

**Discussion by kellijohnson-NOAA** [pfmc-assessments/sa4ss#89] [Show and tell]
![image](https://user-images.githubusercontent.com/4108564/226585500-3c4f0a4a-3cf0-479f-a685-cc3e1fc7ef26.png)

[The styles repository](https://github.com/citation-style-language/styles) on [GitHub](www.github.com) has over 1,000 .csl files for public use and is the official GitHub repository for [csl](https://citationstyles.org/). Previously, @seananderson had made me aware of [csl files stored on zotero](https://www.zotero.org/styles) and [an editor for csl files](https://editor.citationstyles.org/visualEditor), which are also very helpful. [csl](citationstyles.org) mentions users should **[star](https://help.github.com/articles/about-stars/)  our [styles repository](https://github.com/citation-style-language/styles) (and any other of our repositories you find interesting) on GitHub**. So, feel free to star their repository and use the Canadian Journal of Fisheries and Aquatic Sciences csl file that is available in the {sa4ss} repository.

For users that have already ran `draft()` and are working on their assessment documents. Just follow these directions
1. navigate to [the stored csl file](https://github.com/pfmc-assessments/sa4ss/blob/ef4a84d09e7ac5198af6a2d79a120f1360f78153/inst/rmarkdown/templates/sa/skeleton/canadian-journal-of-fisheries-and-aquatic-sciences.csl) and download it,
2.  move the csl file to your bookdown folder, and
3. add the following line to your yaml header in 00a.Rmd that looks like this
```
csl: canadian-journal-of-fisheries-and-aquatic-sciences.csl
```
and your reference section will be formatted to look exactly like a journal article published in the *Canadian Journal of Fisheries and Aquatic Sciences*. I am happy to change this to a different style if we all agree that a different style is needed.

**Comment by iantaylor-NOAA** [comment]
Thank you @kellijohnson-NOAA.
CJFAS style is my favorite style (which is saying something because in general I hate everything about reference sections).
Your instructions worked great to apply CJFAS formatting to the already-drafted petrale skeleton document.
